core/client/mutations/update-wishlist.ts

```diff
@@ -0,0 +1,43 @@
+import { getSessionCustomerId } from '~/auth';
+
+import { client } from '..';
+import { graphql, VariablesOf } from '../graphql';
+
+const UPDATE_WISHLIST_MUTATION = graphql(`
```

Comment:
```suggestion
const UpdateWishlistMutation = graphql(`
```

Reply:
I will use your suggestion in next pr where all wishlist mutations. For now prefer to use current name for it to be consistent with other mutations